spec/factories/users.rb

```diff
@@ -1,8 +1,11 @@
 FactoryBot.define do
   factory :user do
-    last_name { 'MyString' }
-    first_name { 'MyString' }
-    email { 'MyString' }
-    phone { 'MyString' }
+    last_name { 'Last_Name' }
+    first_name { 'First_Name' }
+    email { 'user@example.org' }
+    phone { '+380000000000' }
+    password { 'secret' }
+    password_confirmation { password }
+    project_id { 1 }
```

Comment:
Do not hard code the IDs. Read about association creating here: https://qiita.com/Kolosek/items/7c85337f928161e0e76e

Reply:
> Do not hard code the IDs. Read about association creating here: https://qiita.com/Kolosek/items/7c85337f928161e0e76e

And this factory was merged with a master branch a few hours ago.
Good, I'll change it.